src/nvim/fileio.c

```diff
@@ -6552,6 +6545,7 @@ aucmd_prepbuf (
   /* Allocate "aucmd_win" when needed. */
   if (win == NULL && aucmd_win == NULL) {
     win_alloc_aucmd_win();
+    auwin_appended = true;
```

Comment:
@mhinz what is the name `auwin_appended` trying to convey? That `win_alloc_aucmd_win` implicitly does `win_append`, so it should not be done again below?

Reply:
A belated "yes". :)